src/rsz/include/rsz/Resizer.hh

```diff
@@ -739,6 +749,9 @@ class Resizer : public dbStaState
 
   dpl::Opendp* opendp_ = nullptr;
 
+  std::map<Corner*, std::ofstream> spef_file;
+  bool write_spef_file;
```

Comment:
Data members should have a trailing underscore (https://google.github.io/styleguide/cppguide.html#Variable_Names)